packages/calypso-color-schemes/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/calypso-color-schemes",
-	"version": "2.1.1",
+	"version": "2.2.0",
```

Comment:
Minor version bump should be enough for this change →  `2.1.2`.

Reply:
Done in 5483ec93a467ee8f3675821386635dfe25af79ca